dashboard/src/types/tree/TreeDetails.tsx

```diff
@@ -146,7 +163,8 @@ export const zTreeInformation = z
     commitName: z.string().optional().catch(''),
```

Comment:
another ticket for you, create a ticket to remove parameters that does not need to be in the URL

the URL is to remember user choices

Reply:
Done https://github.com/kernelci/dashboard/issues/671